sklearn/utils/_metadata_requests.py

```diff
@@ -1530,7 +1530,10 @@ def process_routing(_obj, _method, /, **kwargs):
         # an empty dict on routed_params.ANYTHING.ANY_METHOD.
         class EmptyRequest:
             def get(self, name, default=None):
-                return default if default else {}
+                if not default:
+                    return Bunch(**{method: dict() for method in METHODS})
+
+                return default
```

Comment:
codecov is not happy here. I need to figure out when is it the case that `default=None`

Reply:
@adrinjalali I assume that we should be able to cover this one because it would be equivalent to call e.g.

```python
routed_params = _process_routing(self, "score", **kwargs)
routed_params.get("score", default="default")
```

I don't where is the best place to test this. This looks like a metadata routing test to me.